lepton/qemu.go

```diff
@@ -236,8 +253,8 @@ func (q *qemu) setConfig(rconfig *RunConfig) {
 	if rconfig.Bridged {
 		devType = "tap"
 		ifaceName = rconfig.TapName
-		q.addFlag("-enable-kvm")
 	}
+	q.addAccel()
```

Comment:
as much as I'd like to this prob. shouldn't be default w/out having a better install exp to setup permissions correctly - your average user won't know that this takes permissions to run and the average box is not setup to allow access to kvm as non-root

Reply:
idk if macos cares about that or not w/their native hypervisor framework - if not then totally allow it for default on mac

Reply:
`macos` doesn't seem to need additional added user permissions; AFACT  `qemu` from `brew` is not sandboxed and thus doesn't need the added entitlement. Therefore nothing extra is required from the user for the `-accel hvf -cpu host` flags to be added. 

For the other supported systems the install can add the user to the `kvm`. Ops can also skip adding the `-accel kvm` flag if it finds it was run by a user that is not in the `kvm` group and perhaps issue a message notifying the user to add herself to the group.

Reply:
we'll need to add the group functionality to the install script first before putting this in

Reply:
Yes, the group functionality was added. If the installer detects that the current user is not in the `kvm` group it asks to add the user. If "yes" is chosed, it will notify the user that it is using `sudo` to add him to the `kvm` group. If successful in adding the user the installer notifies the user that the changes will take affect after a subsequent login. This is implemented by the `sh` function [op_user_permissions](https://github.com/nanovms/ops/blob/d7fba40ffc1e600f16ab8569d2aa94dfd2aca318/install.sh#L204-L251) in `install.sh`.